.github/workflows/nightly-build.yml

```diff
@@ -8,4 +8,5 @@ on:
 
 jobs:
   nightly:
+    if: github.repository == 'quic-yocto/meta-qcom-hwe'
```

Comment:
we have this if statement in the downstream workflow already, I thought they would apply here. can you describe what's happening without this change?

Reply:
I've been watching this on mine fork but after your question I now saw that it is not updated and maybe that is the cause of my problem.
I synchronized it again now, if the problem persists tomorrow I leave it open, otherwise I close the pr.